app/process-image.js

```diff
@@ -46,14 +47,21 @@ class ImageProcessing {
 		};
 
 		this.gym_pokemon_tesseract_options = Object.assign({}, this.base_tesseract_options, {
-			load_number_dawg: '0',
-			load_punc_dawg: '0'
+			load_number_dawg: '0'
 		});
 
 		this.time_tesseract_options = Object.assign({}, this.base_tesseract_options, {
+			load_system_dawg: '0',
+			tessedit_pageseg_mode: '7',	// character mode; instead of word mode
+			tessedit_char_whitelist: '0123456789:! APM',
```

Comment:
why is "!" a character?

Reply:
I think I forgot to pull that back out - before tuning some other settings, it was wanting to see colons as exclamation points so I'd put that in there since it was better than refusing to recognize the symbol at all, but now even with that as one of the currently-whitelisted characters, it basically never does.

OTOH, it does have a thing on occasion for thinking a time's colon is a 1 or 2, even with pretty high confidence on that (very odd I know), which is why there's a hacky regex in there to detect such a scenario and try replacing that with a colon for purposes of time interpretation.